src/routes/(auth)/+page.svelte

```diff
@@ -50,11 +52,12 @@
       removeable={phrase.user_id === user_id}
       liked={phrase.likes.some((like) => like.user_id === user_id)}
       likeCount={phrase.likes.length}
-      on:cancel={() => deletePhrase(phrase.id)}
-      on:like={() => likedPhrase(phrase.id)}
-      on:unlike={() => unLikePhrase(phrase.id)}
+      on:cancel={async () => await deletePhrase(phrase.id)}
+      on:like={async () => await likedPhrase(phrase.id)}
+      on:unlike={async () => await unLikePhrase(phrase.id)}
```

Comment:
**chore:** it doesn't have much sense defining async function only with an await

Doesn't this have the same effect if rewrite as follows?
```suggestion
      on:cancel={async () => deletePhrase(phrase.id)}
      on:like={async () => likedPhrase(phrase.id)}
      on:unlike={async () => unLikePhrase(phrase.id)}
```

Reply:
Yes